aflr/helper_classes.py

```diff
@@ -60,3 +63,27 @@ def download(cls, scriptId, section=None, parameters=None, destination="."):
             local_filenames.append(local_filename)
 
         return local_filenames
+
+
+class UploadableResource(APIRequest):
```

Comment:
can you rename to upload?

Reply:
I am not sure github is referencing the line I intend to. I mean `upload_audiofile()` -> `upload()`

Reply:
done